views/templates/panes/cfg.pug

```diff
@@ -9,7 +9,7 @@
       .dropdown-menu
         button.btn.btn-sm.btn-light.dropdown-item.export-png(title="Export graph" aria-label="Export graph as an image")
           span.fas.fa-image
-          span.hideable Export PNG
+          span.hideable Export PNG <span class="estimated-export-size"></span>
```

Comment:
Not sure if this is idiomatic pug

Reply:
I think indenting another line under works?

Reply:
```suggestion
          span.hideable Export PNG
            span.estimated-export-size
```

not sure it's better, but it's more pug-gy

Reply:
Oh yeah let's do that